### PR TITLE
Updating Banner Test

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
Banner variant copy increased revenue by 20% so we are switching it to the default and testing it against a new variant. We are also removing the "Learn more" button as part of this.

## Screenshots
New Control:
FebruaryMomentBanner2_control

New Variant:
FebruaryMomentBanner2_variant

## What is the value of this and can you measure success?
This is an A/B test so yes, we will know if it is a success.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [X] Other (please specify): RR/Contributions

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [X] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [X] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [X] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by johnduffell** [comment]
don't forget to delete any parts of the PR template that are not relevant (so that reviewers can see which parts to read)

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss [hunk 1]
```diff
@@ -478,12 +478,7 @@ $wide-circle-height: 162px;
         color: $brightness-100;
         border:  $brand-main 1px solid;
     }
-
-    .engagement-banner__button.engagement-banner__button__learn-more {
```
Comment (johnduffell) [inline static/src/stylesheets/module/site-messages/_fiv-banner.scss]: there are 2 more occurrences in this file that should probably be removed

Comment (johnduffell) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: of `engagement-banner__button__learn-more`

**Review by johnduffell — APPROVED** [review]
thanks for doing this, just a few small comments but after that it looks good! 👍

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss (hunk 1 above)
Comment (jlieb10) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: ✅

**Comment by johnduffell** [comment]
👍

**Pull request merged**